client/reader/list-manage/tag-item.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import { Button } from '@automattic/components';
+import SitePlaceholder from 'blocks/site/placeholder';
+import { Item, Tag } from './types';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+export default function TagItem( props: { item: Item; onRemove: ( e: MouseEvent ) => void } ) {
+	const tag: Tag = props.item.meta.data?.tag?.tag as Tag;
+
+	return ! tag ? (
+		// TODO: Add support for removing invalid tag list item
+		<SitePlaceholder />
+	) : (
+		<>
+			<div className="tag-item list-item">
+				<a className="list-item__content" href={ `/read/tag/${ tag.slug }` }>
```

Comment:
this may need to be url encoded. try an emoji or japanese tag. @bluefuton can probably find ya one

Reply:
Yep! Here are a couple of emoji ones:

https://wordpress.com/tag/%e2%9d%a4%ef%b8%8f (❤️)

https://wordpress.com/tag/%f0%9f%90%b6 (🐶)